lib/ui/painting.dart

```diff
@@ -1892,6 +1892,12 @@ class Path extends NativeFieldWrapperClass2 {
   Path() { _constructor(); }
   void _constructor() native 'Path_constructor';
 
+  // Workaround for tonic, which expects classes with native fields to have a
+  // private constructor.
+  // TODO(dnfield): rework this to use ClaimNativeField - https://github.com/flutter/flutter/issues/50997
+  @pragma('vm:entry-point')
+  Path._() { _constructor(); }
```

Comment:
Super nitty, but can't this be re-written as the following?

```
Path._() => _constructor();
```

Reply:
I suppose - I'll update it.  I'll have another patch coming where this will just become `Path._();`.

Reply:
Actually - this matches the style of a few lines above.  I'd rather just keep it consistent.